### DISCUSSION TITLE
How to read a website database into R?

### DISCUSSION THREAD

**Discussion by Saupia** [teamalgoritma/community#391] [Q&A] [answered]
Hi Algo Data Science Experts, 

I am trying to have open options for doing data visualization, and one of them is a database that I have contributed to for almost 3 years. 
I am trying to get some insight into this website database, is there a way of pulling this website into R?
I see @onlyphantom demonstrated it in class but don't know how to operate it. 
Thank you.

**Comment by kevbow** [comment, marked as answer]
Dear @Saupia,

We are so sorry for the super late reply. 
As for your question, RStudio can access file in a database from the connection pane or write the code to connect it manually. You can check the way to do that from this [Posit link](https://posit.co/blog/working-with-databases-and-sql-in-rstudio/). 
I hope this can help your problem. Thank you!

NB: Posit is the new company of RStudio.
